lib/ohai/plugins/linux/network.rb

```diff
@@ -318,6 +318,10 @@ def parse_ip_addr_inet6_line(cint, iface, line)
     counters Mash.new unless counters
     counters[:network] = Mash.new unless counters[:network]
 
+    network_hint = hint?('network') || {}
+    default_route_table = network_hint.fetch('default_route_table', 'main')
+    Ohai::Log.debug("default route table is '#{default_route_table}'")
```

Comment:
There is a way for users to specify configurable values for plugins in their configuration files, which eliminates the need for hints files in cases like this. Here's an example of a plugin which uses configuration: https://github.com/chef/ohai/blob/081589066eefa00a60192f8cac8801e3b2b31bfe/lib/ohai/plugins/solaris2/filesystem.rb#L66-L74